### PR TITLE
WEB3-521

### PR DESCRIPTION
**Pull request by davehorizen**
Previously empty responses for called to _api_ endpoint with `module=block` and `action=getblockreward` were not managed and the response was this:
```
{
    "message": "Something went wrong.",
    "result": null,
    "status": "0"
}
```
with this PR the response will be:
```
{
    "message": "No block reward data related to this block",
    "result": null,
    "status": "0"
}
```

However the `block_rewards` table of the sidechain database, from which the block reward data is read, doesn't have any record. This will require an additional investigation.

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/block_controller.ex [hunk 1]
```diff
@@ -5,13 +5,18 @@ defmodule BlockScoutWeb.API.RPC.BlockController do
   alias Explorer.Chain
   alias Explorer.Chain.Cache.BlockNumber
 
+  require Logger
```
Comment (yurikoinaba) [inline apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/block_controller.ex:8]: This isn't needed

**Review by yurikoinaba — APPROVED** [review]

**Review by yurikoinaba — COMMENTED** [review]
Can you also add a unit test for when the emission reward table is empty

File: apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/block_controller.ex (hunk 1 above)
Comment (davehorizen) [reply apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/block_controller.ex]: removed
